rocket/urls.py

```diff
@@ -1,13 +1,14 @@
 from django.conf.urls import url
 from . import views
 from django.contrib.auth.views import login,logout
-from .views import grievances,Profile
+from .views import grievances,Profile,UserProfileEdit
 
 urlpatterns=[
     url(r'^login/$', login, {'template_name':'rocket/login.html'}),
     url(r'^signup/$', views.signup.as_view(), name='signup'),
     url(r'^profile/$', Profile.as_view(), name='profile'),
     url(r'^edit_profile/$', views.edit_profile, name='edit_profile'),
+     url(r'^userprofile_edit/$',UserProfileEdit.as_view(), name='userprofile_edit' ),
```

Comment:
By using edit_profile user can change his account info like first_name,last_name,username.
User account was created when user first signup to the website.
And by userprofile_edit  user can change his bio,phone,etc..we can add  more fields in this form if we want some more info about user.